behat/tests/bootstrap/v2/common/ElementSelectionTrait.php

```diff
@@ -0,0 +1,93 @@
+<?php declare(strict_types=1);
+
+namespace DigidepsBehat\v2\Common;
+
+trait ElementSelectionTrait
+{
+    public function iClickOnNthElementBasedOnRegex(string $regex, int $elementIndex)
```

Comment:
Nice - could we get a brief comment explaining the usage for all of the functions in this class?

Reply:
Great idea. Yeah will do that